### PR TITLE
Create contact form

### PR DESCRIPTION
**Pull request by rosemutai**
In this section of my portfolio:

- I added a contact form, with name, email, and message inputs.
- I styled these inputs using CSS
- I added HTML form validations
- I added a data collection feature using the Formspree service

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,6 +232,27 @@ <h4 class="connect-link padding">LET'S CONNECT</h4>
...
+          </li>
+          <li>
+            <input type="email" name="email" id="youremail" placeholder="natajayanj14@gmail.com" required>
+          </li>
+          <li>
+            <textarea name="message" id="yourmessage" cols="" rows="5" placeholder="Write your message here" required maxlength="500"></textarea>
+          </li>
+        </ul>
+    
+        <button type="submit">Get in touch</button>
```
Comment (wayungi) [inline index.html]: - [x] Please implement the button interaction states as shown in the mage below.
btns

File: index.html [hunk 2]
```diff
@@ -232,6 +232,27 @@ <h4 class="connect-link padding">LET'S CONNECT</h4>
 
       </section>
     </section>
+    <section class="contact-form-section">
+      <h2 class="contact-me-title">Contact me</h2>
+      <p class="contact-description">If you have an application you are interested in developing,
+        a feature that you need built or a project that needs
+        coding. I’d love to help with it</p>
```
Comment (wayungi) [inline index.html:239]: - [x] Excellent work implementing this section of the form. You still have some slight tweaking to do before we can close this 😄. The design text is centered but yours is not. Please make this fix and get the all clear

**Design**
scrn1
**Your implementation**
scrn2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @rosemutai , 
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @rosemutai,

This is Adnan Is-haq, your beloved code reviewer 😁 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice code organization
- Professional README.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**